ir/node.cpp

```diff
@@ -102,6 +102,17 @@ Util::JsonObject* IR::Node::sourceInfoJsonObj() const {
     return json;
 }
 
+static cstring quote(cstring s) {
```

Comment:
would be nice, because otherwise, nobody is going to find it and it will be bound to repeat. A comment on it would also help :)